py/sight/widgets/decision/decision.py

```diff
@@ -181,7 +181,7 @@
     ['gcs', 'local', 'redis', 'none', 'gcs_with_redis', 'local_with_redis'],
     'Which Sight cache to use ? (default is none)')
 
-_file_name = 'decision_actor.py'
+_file_name = 'decision.py'
```

Comment:
Do we still need this? If yes, can we replace it with __file__ or os.path.basename(__file__)?